docs/usage/PRODUCT_SYNC.md

```diff
@@ -86,11 +86,11 @@ events.
 
 - `beforeUpdateCallback`
 a filter function which can be applied on a generated list of update actions. It allows the user to intercept product 
-update and modify (add/remove) update actions just before they are send to CTP API.
+update and modify (add/remove) update actions just before they are sent to CTP API.
 
 - `beforeCreateCallback`
 a filter function which can be applied on a product draft before a request to create it on CTP is issued. It allows the 
-user to intercept product create requests modify the draft before the create request is sent to CTP API.
+user to intercept product creates requests modify the draft before the create request is sent to CTP API.
```

Comment:
check the same comment in the previous doc file. (CATEGORY_SYNC.md).

Reply:
https://github.com/commercetools/commercetools-sync-java/pull/319/commits/7294c540f5cba7f6cc84fdef6ca894f1e801694d

Reply:
nitpick: maybe make create requests bold and/or italic -> create requests as well as update requests